[2021-05-07 15:23:44,346] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: bash_dag.print_date 2020-05-02T18:52:00+00:00 [queued]>
[2021-05-07 15:23:44,702] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: bash_dag.print_date 2020-05-02T18:52:00+00:00 [queued]>
[2021-05-07 15:23:44,703] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-05-07 15:23:44,703] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-05-07 15:23:44,703] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-05-07 15:23:44,762] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): print_date> on 2020-05-02T18:52:00+00:00
[2021-05-07 15:23:44,798] {standard_task_runner.py:52} INFO - Started process 5389 to run task
[2021-05-07 15:23:44,831] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'bash_dag', 'print_date', '2020-05-02T18:52:00+00:00', '--job-id', '17', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/test_dag1.py', '--cfg-path', '/tmp/tmph4an3jl0', '--error-file', '/tmp/tmpk6p1weoi']
[2021-05-07 15:23:44,838] {standard_task_runner.py:77} INFO - Job 17: Subtask print_date
[2021-05-07 15:23:45,625] {logging_mixin.py:104} INFO - Running <TaskInstance: bash_dag.print_date 2020-05-02T18:52:00+00:00 [running]> on host jaarora-WX-1.corp.adobe.com
[2021-05-07 15:23:47,295] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=jacrod2901@gmail.com
AIRFLOW_CTX_DAG_OWNER=jacrod2901
AIRFLOW_CTX_DAG_ID=bash_dag
AIRFLOW_CTX_TASK_ID=print_date
AIRFLOW_CTX_EXECUTION_DATE=2020-05-02T18:52:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-05-02T18:52:00+00:00
[2021-05-07 15:23:47,296] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-05-07 15:23:47,298] {bash.py:158} INFO - Running command: date
[2021-05-07 15:23:47,333] {bash.py:169} INFO - Output:
[2021-05-07 15:23:47,358] {bash.py:173} INFO - Fri May  7 15:23:47 IST 2021
[2021-05-07 15:23:47,359] {bash.py:177} INFO - Command exited with return code 0
[2021-05-07 15:23:47,762] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=bash_dag, task_id=print_date, execution_date=20200502T185200, start_date=20210507T095344, end_date=20210507T095347
[2021-05-07 15:23:48,262] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-05-07 15:23:48,316] {local_task_job.py:146} INFO - Task exited with return code 0
